modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/components/sequence/steps/StepsTable.scala

```diff
@@ -165,28 +169,36 @@ object StepsTable {
 
     // Columns for the table
     private def columns(p: Props, s: Size): List[Table.ColumnArg] = {
-      val (offsetVisible, exposureVisible, fpuVisible, filterVisible, objectSize) = s.width match {
-        case w if w < PhoneCut      => (false, false, false, false, SSize.Tiny)
-        case w if w < LargePhoneCut => (false, true, false, false, SSize.Small)
-        case _                      => (displayOffsets(p), true, true, true, SSize.Small)
+      val (offsetVisible, exposureVisible, disperserVisible, fpuVisible, filterVisible, objectSize) = s.width match {
+        case w if w < PhoneCut      => (false, false, false, false, false, SSize.Tiny)
+        case w if w < LargePhoneCut => (false, true, false, false, false, SSize.Small)
```

Comment:
Did you test on a narrower screen? This part of the code will hide columns when we can't display all of them, it would be good to verify how it looks on smaller screen (Responsive mode)

Reply:
Here's a screenshot. I'm assuming that this is what we want, based on the other parameters that are hidden when real estate is small?
![screen shot 2018-02-21 at 4 27 55 pm](https://user-images.githubusercontent.com/8795653/36500906-528aadbe-1724-11e8-9567-4d64a027c54d.png)

Reply:
Thanks, that seems correct

Reply:
Oh, that was my other question, though: is this the position that we want the disperser column in in the table? I don't know.

Reply:
Can you ask @andrewwstephens about that?

Reply:
The order looks good to me.